charts/super-agent-deployment/README.md

```diff
@@ -36,10 +36,12 @@ At the point of the creation of the chart, it has no particularities and this se
 |-----|------|---------|-------------|
 | affinity | object | `{}` | Sets pod/node affinities. Can be configured also with `global.affinity` |
 | cluster | string | `""` | TODO: Name of the Kubernetes cluster monitored. Can be configured also with `global.cluster`. |
-| config.content | string | See `values.yaml` for examples | Here you can set New Relic' Super Agent configuration. |
-| config.create | bool | `true` | Set if the configMap is going to be created by this chart or the user will provide its own. |
-| config.key | string | `""` | The key in the configMap that has the configuration for the Super Agent. |
-| config.name | string | `""` | The name the configMap is going to have. If create is set to false, the name of an existing configMap that will be used to config the Super Agent. |
+| config.fleet | object | See `values.yaml` for examples | Values that the fleet is going to have in the deployment. |
```

Comment:
I believe you are missing to generate again the docs

Reply:
both for fleet and "content"

Reply:
Done on [#c7436bf](https://github.com/newrelic/helm-charts/pull/1221/commits/c7436bf5473024ce7a3034236bc97169695ae943)

The part of the `content` is not on the `helm-docs` because it is a comment (yet).